src/components/lab/EmittedLine.js

```diff
@@ -74,8 +75,8 @@ export default class EmittedLine extends Component {
 
       // the first point is where the charge is
       // add second point where the new point should be
-      // keeps only MAX_POINTS_FOR_LINES first points
-      newPoints = [0, 0, x, y, ...newPoints].slice(0, MAX_POINTS_FOR_LINES);
+      // keeps only the maxPointsForLine first points, sufficient to fill screen
+      newPoints = [0, 0, x, y, ...newPoints].slice(0, maxPointsForLines);
```

Comment:
I just had to do it: maybe make sure slicing the array given this `maxPointForLines` number will result in an even-sized array. I just had strange behaviour because of this issue.